internal/server/create_document_digital_lpa.go

```diff
@@ -120,6 +182,12 @@ func CreateDocumentDigitalLpa(client CreateDocumentDigitalLpaClient, tmpl templa
 					continue
 				}
 
+				// Create contact for each recipient as we currently have no way to link a document to LPA store actors
+				recipient, err = client.CreateContact(ctx, recipient)
+				if err != nil {
+					return err
+				}
```

Comment:
Can we wrap this in `if (recipient.ID < 0)` to avoid creating new records for recipients that already exist? I appreciate that can't happen right now, but it will be useful in the future if we ever populate `data.Recipients` with existing people records

Reply:
Good point this is done.